VideoLocker/src/main/java/org/edx/mobile/authentication/LoginService.java

```diff
@@ -0,0 +1,26 @@
+package org.edx.mobile.authentication;
+
+import org.edx.mobile.http.ApiConstants;
+
+import retrofit.http.Field;
+import retrofit.http.FormUrlEncoded;
+import retrofit.http.POST;
+
+/**
+ * Created by cleeedx on 4/11/16.
+ */
+public interface LoginService {
+
+    @FormUrlEncoded
+    @POST(ApiConstants.URL_ACCESS_TOKEN)
+    AuthResponse getAccessToken(@Field("grant_type") String grant_type,
```

Comment:
Just to confirm, these two requests really are just the same endpoint with different parameters?